### PR TITLE
Allow installation under PHP 8.x

### PR DESCRIPTION
**Pull request by cwittstock**
Its not possible to run this extension with contao 4.13 and PHP8

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     ],
     "require": {
         "ext-json": "*",
-        "php": ">=7.1",
+        "php": ">=7.1||^8.0",
```
Comment (bytehead) [inline composer.json]: Because of consistency with the rest of the `composer.json` file:

```suggestion
        "php": ">=7.1 || ^8.0",
```

**Review by bytehead — CHANGES_REQUESTED** [review]

**Comment by bytehead** [comment]
Thank you - I'm not sure if its installable after, as the MailChimp API wrapper still uses Guzzle 6. Possibly compatible with PHP 8 😬

**Comment by netzarbeiter** [comment]
## [Version Guidance](https://github.com/guzzle/guzzle)

| Version | Status         | Packagist           | Namespace    | Repo                | Docs                | PSR-7 | PHP Version  |
|---------|----------------|---------------------|--------------|---------------------|---------------------|-------|--------------|
| 3.x     | EOL            | `guzzle/guzzle`     | `Guzzle`     | [v3][guzzle-3-repo] | [v3][guzzle-3-docs] | No    | >=5.3.3,<7.0 |
| 4.x     | EOL            | `guzzlehttp/guzzle` | `GuzzleHttp` | [v4][guzzle-4-repo] | N/A                 | No    | >=5.4,<7.0   |
| 5.x     | EOL            | `guzzlehttp/guzzle` | `GuzzleHttp` | [v5][guzzle-5-repo] | [v5][guzzle-5-docs] | No    | >=5.4,<7.4   |
| 6.x     | Security fixes | `guzzlehttp/guzzle` | `GuzzleHttp` | [v6][guzzle-6-repo] | [v6][guzzle-6-docs] | Yes   | >=5.5,<8.0   |
| 7.x     | Latest         | `guzzlehttp/guzzle` | `GuzzleHttp` | [v7][guzzle-7-repo] | [v7][guzzle-7-docs] | Yes   | >=7.2.5,<8.2 |

Guzzle 6 is and will not be compatible with PHP 8.

I have unfortunately not found any information on the further development of the API of Mailchimp.

Do you see any other way to keep this extension alive or would you then advise against using it?

**Comment by bytehead** [comment]
> Guzzle 6 is and will not be compatible with PHP 8.

It actually works under PHP 8 but there is no official clearance from the developers 😅 

@netzarbeiter did you try out to install and run this patch/branch?

**Review by bytehead — APPROVED** [review]

**Comment by bytehead** [comment]
Thank you @cwittstock!

**Pull request merged**